programs/dasheri/src/pool/instructions/create_mango_account.rs

```diff
@@ -25,6 +31,8 @@ pub struct PoolCreateMangoAccount<'info> {
     pub system_program: Program<'info, System>,
 }
 
+// REVIEW: so there could be multiple mango_accounts per pool?
```

Comment:
technically yes, you can use the account_num to do that, of course if you use the same pda it would crash, I would prefer keeping it separate, since the number of accounts going to the ix blows up and becomes hard to read (did try it just now and got really hard to follow)